app/models/scale.rb

```diff
@@ -0,0 +1,35 @@
+class Scale < ScoredItem
+    include QuestionHelper
+  
+    attr_accessor :txt, :type, :weight, :min_label, :max_label, :answer, :min_question_score, :max_question_score
+  
+    def edit
+      edit_common('Item:', min_question_score, max_question_score , txt, weight, type).to_json
+    end
+  
+    def view_question_text
```

Comment:
--> view_item_text (and ditto for other files, if any of them say "view_question_text").